src/test/java/com/example/springassignment3/TestService.java

```diff
@@ -0,0 +1,73 @@
+package com.example.springassignment3;
+
+import com.example.springassignment3.repository.TestRepository;
+import com.example.springassignment3.service.TestServiceImpl;
+import com.example.springassignment3.entity.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Optional;
+
+@SpringBootTest
+public class TestService
+{
+    @InjectMocks
+    private TestServiceImpl testService;
+
+    @Mock
+    private TestRepository testRepository;
+
+    @BeforeEach()
+    void setup() throws Exception
+    {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @org.junit.jupiter.api.Test
```

Comment:
Use @Test mock directly. Move this to imports

Reply:
Actually I have entity with the same name , that is why need to use it this way